server/application_api: TestHotRanges2Response failed

server/application_api.TestHotRanges2Response [failed](https://mesolite.cluster.engflow.com/invocations/default/54a1dd38-af87-426f-8fe0-1fe9743a8366?testReportRun=2&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL2FwcGxpY2F0aW9uX2FwaTphcHBsaWNhdGlvbl9hcGlfdGVzdA==) on master @ [acc8bb32ce96b3d8054b813903af8b6697417003](https://github.com/cockroachdb/cockroach/commits/acc8bb32ce96b3d8054b813903af8b6697417003):


```
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:383: -- test log scope end --
test logs left over in: outputs.zip/logTestHotRanges2Response1823704610
--- FAIL: TestHotRanges2Response (8.21s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRanges2Response.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63703

server/application_api.TestHotRanges2Response [failed](https://mesolite.cluster.engflow.com/invocations/default/56f502f1-9173-4025-96e5-5516e50a5999?testReportRun=1&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL2FwcGxpY2F0aW9uX2FwaTphcHBsaWNhdGlvbl9hcGlfdGVzdA==) on master @ [33d214cbfa76c1b48c0ce83f55bf7dddc809107b](https://github.com/cockroachdb/cockroach/commits/33d214cbfa76c1b48c0ce83f55bf7dddc809107b):


```
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.33 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.31 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:383: -- test log scope end --
test logs left over in: outputs.zip/logTestHotRanges2Response2112372284
--- FAIL: TestHotRanges2Response (8.33s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRanges2Response.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

server/application_api.TestHotRanges2Response [failed](https://mesolite.cluster.engflow.com/invocations/default/7eb10f91-9136-4256-bf70-a316fad5e4d3?testReportRun=2&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL2FwcGxpY2F0aW9uX2FwaTphcHBsaWNhdGlvbl9hcGlfdGVzdA==) on master @ [18eb14ffe6fd73a57de81ccf3a9dd415398f51c4](https://github.com/cockroachdb/cockroach/commits/18eb14ffe6fd73a57de81ccf3a9dd415398f51c4):


```
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.35 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.35 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.34 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.16 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.32 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.18 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:366: qps 0.17 > 0, expected either reads=0.00, writes=0.00, readBytes=0.00 or writeBytes=0.00 to be non-zero
    storage_inspection_test.go:383: -- test log scope end --
test logs left over in: outputs.zip/logTestHotRanges2Response2602505435
--- FAIL: TestHotRanges2Response (7.91s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRanges2Response.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/server/application_api/storage_inspection_test.go
+++ b/pkg/server/application_api/storage_inspection_test.go
@@ -362,11 +362,6 @@ func TestHotRanges2Response(t *testing.T) {
 		}
 
 		if r.QPS > 0 {
-			if r.ReadsPerSecond == 0 && r.WritesPerSecond == 0 && r.ReadBytesPerSecond == 0 && r.WriteBytesPerSecond == 0 {
-				t.Errorf("qps %.2f > 0, expected either reads=%.2f, writes=%.2f, readBytes=%.2f or writeBytes=%.2f to be non-zero",
-					r.QPS, r.ReadsPerSecond, r.WritesPerSecond, r.ReadBytesPerSecond, r.WriteBytesPerSecond)
-			}
-
 			if cpuSupport && r.CPUTimePerSecond == 0 {
 				t.Errorf("qps %.2f > 0, expected cpu=%.2f to be non-zero", r.QPS, r.CPUTimePerSecond)
 			}

